setup.py

```diff
@@ -6,16 +6,18 @@
 INSTALL_REQUIREMENTS = [
     "Django>=1.11,<3.0",
     "django-treebeard>=4.3",
-    "django-cms"
+    "django-cms",
 ]
 
 TEST_REQUIREMENTS = [
+    "beautifulsoup4",
     "djangocms_helper",
     "djangocms_versioning",
     "djangocms_version_locking",
     "djangocms-moderation",
     "factory_boy",
-    "django_cms"
+    "lxml",
```

Comment:
lxml and BS4? Personally I would go with lxml, it's smaller and faster in every benchmark I've ever seen.